src/components/LineChart.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] [OPTIONAL] Linting is a great tool to make your code clean so please avoid disabling them

Reply:
Thank you @hiromataba